.gitignore

```diff
@@ -18,3 +18,4 @@ cabal.sandbox.config
 *.hp
 *.eventlog
 .stack-work/
+stack.yaml
```

Comment:
I thought this might make sense since .stack-work was already in the .gitignore, but I don't mind removing it.